packages/core/src/components/dialog/_multistep-dialog.scss

```diff
@@ -0,0 +1,86 @@
+// Copyright 2020 Palantir Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0.
+
+@import "~@blueprintjs/icons/src/icons";
+@import "../../common/mixins";
+@import "../../common/react-transition";
+@import "../../common/variables";
+
+$dialog-border-radius: $pt-border-radius * 2 !default;
+$step-radius: $pt-border-radius * 2 !default;
+
+.#{$ns}-multistep-dialog-panels {
+  display: flex;
+}
+
+.#{$ns}-multistep-dialog-left-panel {
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+}
+
+.#{$ns}-multistep-dialog-right-panel {
+  background-color: $light-gray5;
+  border-left: 1px solid $pt-divider-black;
+  border-radius: 0 0 $dialog-border-radius 0;
+  flex: 2;
+}
+
+.#{$ns}-multistep-dialog-footer {
+  background-color: $white;
+  border-radius: 0  0 $dialog-border-radius 0;
+  border-top: 1px solid $pt-divider-black;
+  padding: 10px;
+}
+
+.#{$ns}-step-container {
+  background-color: $light-gray5;
+  border-bottom: 1px solid $pt-divider-black;
+
+  &.#{$ns}-active {
+    background-color: $white;
+  }
+}
+
+.#{$ns}-step {
+  align-items: center;
+  background-color: $light-gray5;
+  border-radius: $step-radius;
+  cursor: not-allowed;
```

Comment:
On further review, I realized the semantics of "active" here are a little different from what I expected. That's fine, I think what you have works. but I think it would help to leave a code comment along the lines of "by default, steps are inactive until they are visited"